ts/src/test/static/request/gate.json

```diff
@@ -392,6 +392,20 @@
                     "LTC/USDT"
                 ]
             },
+            {
```

Comment:
is it possible to add swap+trigger tests?

Reply:
> is it possible to add swap+trigger tests?

pushed atm

Reply:
@ttodua hum I can't see them here

Reply:
> @ttodua hum I can't see them here

![image](https://github.com/user-attachments/assets/f32395fe-5f4b-44e2-90d6-9ce52ef34c60)

i was referring to this one, do you mean smth other?

Reply:
@ttodua do we have trigger+swap fetchOpenOrders static request tests?